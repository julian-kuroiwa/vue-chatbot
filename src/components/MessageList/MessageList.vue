<template>
	<div class="message-list">
		<slot></slot>
	</div>
</template>

<style>
	.message-list {
		height: calc(100vh - 60px);
		padding: 10px;
	}

	.message-list p {
		margin: 0;
	}
</style>

<script>
	export default {
		name: 'MessageList'
	}
</script>
