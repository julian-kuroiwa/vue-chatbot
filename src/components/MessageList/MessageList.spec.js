import { mount } from '@vue/test-utils';
import MessageList from './MessageList';

describe('<MessageList />', () => {
	const wrapper = mount(MessageList, {
		propsData: {
			messages: [
				{
					message: 'teste',
				},
				{
					message: 'teste teste',
					user: true
				}
			]
		}
	})

	it('Component is rendered', () => {
		expect(wrapper.props()).toEqual({})
	});

	it('renders correctly', () => {
		const wrapper = mount(MessageList)
		expect(wrapper.element).toMatchSnapshot()
	});
});
