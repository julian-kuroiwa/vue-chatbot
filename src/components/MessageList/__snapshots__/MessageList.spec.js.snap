// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MessageList /> renders correctly 1`] = `
<div
  class="message-list"
/>
`;
